crates/js-sys/tests/wasm/Array.rs

```diff
@@ -466,3 +466,8 @@ fn array_inheritance() {
     assert!(array.is_instance_of::<Object>());
     let _: &Object = array.as_ref();
 }
+
+#[wasm_bindgen_test]
+fn uint8_view_mut_raw() {
+    assert!(1 == 2);
```

Comment:
@alexcrichton How do I run the tests locally?
`cargo test -p js-sys` does not run anything:
```
$ cargo test -p js-sys
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running target/debug/deps/headless-cb15791b0fc6e9fb

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

     Running target/debug/deps/wasm-d7d4f9b8001c0920

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
```
```
$ cargo +nightly test --target wasm32-unknown-unknown -p js-sys
   Compiling cfg-if v0.1.10
   Compiling scoped-tls v1.0.0
   Compiling wasm-bindgen-backend v0.2.53 (/home/lshlyapnikov/Projects/rustwasm/wasm-bindgen/crates/backend)
error[E0463]: can't find crate for `core`
  |
  = note: the `wasm32-unknown-unknown` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `cfg-if`.
warning: build failed, waiting for other jobs to finish...
error[E0463]: can't find crate for `std`
  |
  = note: the `wasm32-unknown-unknown` target may not be installed

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `scoped-tls`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

Reply:
You'll want to `rustup target add wasm32-unknown-unknown` I believe